apps/desktop-wallet/src/modals/SendModals/CallContract/BuildTxModalContent.tsx

```diff
@@ -32,13 +32,14 @@ import useGasSettings from '@/hooks/useGasSettings'
 import useStateObject from '@/hooks/useStateObject'
 import AssetAmountsInput from '@/modals/SendModals/AssetAmountsInput'
 import GasSettings from '@/modals/SendModals/GasSettings'
+import { CallContractTxModalData } from '@/modals/SendModals/sendTypes'
```

Comment:
Mhh, what about having this and the modals in `features/send`?

Reply:
YES! I just didn't want to do it yes so that you can inspect the diff! Now that you did a first pass, I'll make the move!

Reply:
Done!